content/blog/2018/01/2018-01-15-overhaul-of-manage-jenkins-page.adoc

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: "Overhaul of Manage Jenkins page"
+tags:
+- jenkins
+- ui
+- restyling
+author: recena
+---
+
+== Overview
+
+Recently we have introduced some UI improvements around to the Manage Jenkins page. The visual changes are very subtle but behind of them, there are interesting benefits.
```

Comment:
A passive voice sounds better ==> "Some UI improvements around to the Manage Jenkins page have been introduced"